submission/ContextTabs/background.js

```diff
@@ -0,0 +1,68 @@
+const extensionAPI = typeof browser !== "undefined" ? browser : chrome;
+
+console.log("Background script running...");
+
+let currentContext = null;
+
+// Predefined groups and sites
+const predefinedContexts = {

```

Comment:
It would be very useful to be able to add a feature to create new tab groups and completely delete existing groups. Please add that in the future and I can see myself personally using that more often... I already really like the "work", "entertainment" and "research" tab groups.